avm/res/network/virtual-network-gateway/main.bicep

```
@@ -179,7 +186,8 @@ var bgpSettingsVar = isActiveActive
       ]
     }
 
-// Potential IP configurations (active-active vs active-passive)
+
+// Potential IP configurations (Active-Active vs active-passive)
```

If change required I guess it should apply also to Active-Passive
```suggestion
// Potential IP configurations (Active-Active vs Active-Passive)
```